File: 프로그래머스/1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    let answer = '';
    const upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
    const lower = "abcdefghijklmnopqrstuvwxyz"
    
    for (let v of s){
        if (v === " ") answer += v
        else {
            let Aa = upper.includes(v)? upper : lower;
            answer += Aa[(Aa.indexOf(v)+n) % 26] 
        }
    }
    return answer;
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    alpha = list("abcdefghijklmnopqrstuvwxyz")
    answer = ""
    
    for v in s:
        if v != " ":
            idx = alpha.index(v.casefold())+n
            if idx > len(alpha)-1: idx %= len(alpha)
        
        if v == " ": answer += v
        elif v in alpha: answer += alpha[idx]
        else: answer += alpha[idx].upper()
        
    return answer